lib/payment/merchant_connector.rb

```diff
@@ -162,7 +162,8 @@ def base_path
 
     # :nodoc:
     module CreateChargeParams
-      REQUIRED_PARAMS = [:credit_card_token, :amount_cents, :currency].freeze
+      REQUIRED_PARAMS = [:credit_card_token, :amount_cents, :currency,
+                         :captured].freeze
```

Comment:
Yes, you're right, they use 'captured' only when thecharge is already created. I'll fix using 'capture' attribute for creating a Charge, but will use 'captured' when it's just a read-only attribute (and will use 'captured' in the db too). What do you think?